diff -Nur unionfs-1.3.orig/Makefile unionfs-1.3.build/Makefile
--- unionfs-1.3.orig/Makefile	2006-06-26 19:46:44.000000000 +0000
+++ unionfs-1.3.build/Makefile	2006-11-13 20:35:25.000000000 +0000
@@ -16,7 +16,7 @@
 SUP_EXTRA=
 TAROPTS = czvf
 COMPEXT = .gz
-UNIONFS_DEBUG_CFLAG = -g
+UNIONFS_DEBUG_CFLAG =
 UNIONFS_OPT_CFLAG= -O2
 UNIONFS_VERSION_CFLAG=-DUNIONFS_VERSION=\"${VERSION}\" -DSUP_MAJOR=${SUP_MAJOR} -DSUP_MINOR=${SUP_MINOR} -DSUP_PATCH=${SUP_PATCH}
 
@@ -33,8 +33,8 @@
 
 CC	= gcc
 KERNELVERSION=$(shell echo $(KVERS) | cut -d. -f1,2)
-EXTRA_CFLAGS += ${TOPINC} -Wall -Werror ${EXTRACFLAGS} ${UNIONFS_DEBUG_CFLAG} ${UNIONFS_OPT_CFLAG} ${UNIONFS_VERSION_CFLAG}
-UCFLAGS = -I. ${UNIONFS_DEBUG_CFLAG} ${UNIONFS_OPT_CFLAG} -Wall -Werror ${EXTRAUCFLAGS}  ${UNIONFS_VERSION_CFLAG}
+EXTRA_CFLAGS += ${TOPINC} -Wall ${EXTRACFLAGS} ${UNIONFS_DEBUG_CFLAG} ${UNIONFS_OPT_CFLAG} ${UNIONFS_VERSION_CFLAG}
+UCFLAGS = -I. ${UNIONFS_DEBUG_CFLAG} ${UNIONFS_OPT_CFLAG} -Wall ${EXTRAUCFLAGS}  ${UNIONFS_VERSION_CFLAG}
 
 obj-m := unionfs.o
 unionfs-objs := subr.o dentry.o file.o inode.o main.o super.o \
@@ -84,7 +84,7 @@
 ${unionfs-objs}: unionfs.h
 
 utils: FRC
-	make -C utils all CC=${CC} UCFLAGS='-I.. ${UCFLAGS}'
+	make -C utils all CC='${CC}' UCFLAGS='-I.. ${UCFLAGS}'
 
 install-utils: utils
 	make -C utils install PREFIX=${PREFIX}
diff -Nur unionfs-1.3.orig/utils/Makefile unionfs-1.3.build/utils/Makefile
--- unionfs-1.3.orig/utils/Makefile	2006-06-26 19:46:45.000000000 +0000
+++ unionfs-1.3.build/utils/Makefile	2006-11-13 20:35:59.000000000 +0000
@@ -4,13 +4,13 @@
 all:	${BINS}
 
 unionctl: unionctl.c usercommon.c
-	${CC} -o $@ $^ ${UCFLAGS}
+	${CC} ${UCFLAGS} -o $@ $^
 
 uniondbg: uniondbg.c
-	${CC} -o $@ $^ ${UCFLAGS}
+	${CC} ${UCFLAGS} -o $@ $^
 
 unionimap: unionimap.c usercommon.c
-	${CC} -o $@ $^ -luuid ${UCFLAGS}
+	${CC} ${UCFLAGS} -o $@ $^ -luuid
 
 TAGS: $(wildcard *.[ch])
 	etags -t $^
